src/app/static/src/js/components/permissions/roleList.vue

```diff
@@ -2,10 +2,9 @@
     <ul class="list-unstyled roles" v-if="roles.length > 0">
         <li v-for="(role, index) in roles"
             v-bind:id="role.name"
-            v-bind:class="['role', {'open':expanded[index]}, {'has-members': role.members.length > 0}]"
-            v-on:click="toggle(index)">
-            <div class="expander"></div>
-            <span v-text="role.name" class="role-name"></span>
+            v-bind:class="['role', {'open':expanded[index]}, {'has-members': role.members.length > 0}]">
+            <div class="expander" v-on:click="toggle(index)"></div>
+            <span v-text="role.name" class="role-name" v-on:click="toggle(index)"></span>
```

Comment:
I moved the click toggle to the div and span as the stopPropagation from the user list didn't appear to be working so any click on the user list would close the role. Not sure why that is, but it does also seem to be affecting the  roleList in the reports page as deployed.